src/lib/mdns/Discovery_ImplPlatform.cpp

```diff
@@ -432,6 +433,26 @@ CHIP_ERROR DiscoveryImplPlatform::ResolveNodeId(const PeerId & peerId, Inet::IPA
 {
     ReturnErrorOnFailure(Init());
 
+    FabricId fabricId;
+    Inet::IPAddress addr;
+    uint16_t port;
+    Inet::InterfaceId iface;
+
+    if (sIPCache.Lookup(peerId.GetNodeId(), fabricId, addr, port, iface) == CHIP_NO_ERROR)
```

Comment:
hmm... If this is the case, how should I interpret this: https://github.com/project-chip/connectedhomeip/blob/d1a9ee9ebbbd1c8f92b7cbb9572c5f01a748ed17/src/lib/mdns/Resolver_ImplMinimalMdns.cpp#L170?

Reply:
The code shows that name == string representation of fabric+node.

So DNS name and peerid are functionally equivalent.

I do not understand the relevance of the referenced code for this line though - I am asking to do a `sIPCache.Lookp(peerId, addr, port, ...)` and not have fabricId in there because rather than saying "I give you the node, you give me fabric and address" this should be"I give you node and fabric, you give me address".

"1-1" and "2-1" are valid mdns names for different nodes, both having nodeid 1 and different fabric.